csharp/Facebook.Yoga/Native.cs

```diff
@@ -20,18 +20,40 @@ internal static class Native
         private const string DllName = "yoga";
 #endif
 
+        internal class YGNodeHandle : SafeHandle
+        {
+            private YGNodeHandle() : base(IntPtr.Zero, true)
+            {
+            }
+
+            public override bool IsInvalid
+            {
+                get
+                {
+                    return this.handle == IntPtr.Zero;
+                }
+            }
+
+            protected override bool ReleaseHandle()
+            {
+                Native.YGNodeFree(this.handle);
+                GC.KeepAlive(this);
```

Comment:
Likely not necessary, but I wasn't sure of the implementation of the parent class. Shouldn't hurt.